### PR TITLE
Several improvements of AutoFrontlight plugin and enable it by default

### PR DESCRIPTION
**Pull request by Hzj-jie**

### CODE REVIEW

**Comment by Frenzie** [comment]
Does this also address #2973?

**Comment by Hzj-jie** [comment]
That has been fixed already.

**Comment by Frenzie** [comment]
Neat!

**Comment by Hzj-jie** [comment]
@Frenzie Some strange behavior happens with this change, the file manager menu refused to show "more_plugins" menu. Have I done something wrong?

**Comment by Frenzie** [comment]
I don't see anything obviously wrong or different but unfortunately I don't have time to test.

**Comment by Hzj-jie** [comment]
It's strange that with this change, Help / Disable dictionary fuzzy search / More show in the settings. No problem, I will debug tomorrow.

**Comment by Hzj-jie** [comment]
A quick question, this issue does not happen in ReaderMenu, do you remember if anything different between reader and file manager?

**Comment by Frenzie** [comment]
The plugins/extensions menu may have a different name in each but from what
I saw in the diff that shouldn't come into play.

**Comment by Hzj-jie** [comment]
It's an issue in MenuSorter: it should retry processing sub menus until nothing changes. Because the order in an Lua table is undefined, and a sub menu cannot be referred until its parent menu is inserted into menu_table["KOMenu:menu_buttons"].

**Comment by Frenzie** [comment]
It's definitely not meant to quietly depend on a certain order regardless of whether Lua messes it up. It was probably an (un)fortunate series of events in which I ordered my tests by some form of logic combined with Lua not changing it.

File: frontend/ui/elements/common_info_menu_table.lua [hunk 1]
```diff
@@ -23,6 +23,9 @@ common_info.version = {
 common_info.help = {
     text = _("Help"),
 }
+common_info.more_plugins = {
+    text = _("More"),
```
Comment (Frenzie) [inline frontend/ui/elements/common_info_menu_table.lua]: Wouldn't "more plugins" also be clearer in the menu itself?

Comment (Hzj-jie) [inline frontend/ui/elements/common_info_menu_table.lua]: Updated.

**Comment by Hzj-jie** [comment]
Yes, I totally agree :) So once the new item has been added, (un)fortunately two items instead of only the newly added one are reordered.

**Pull request merged**